openwisp_notifications/static/openwisp-notifications/js/notifications.js

```diff
@@ -387,6 +387,7 @@ function notificationHandler($, elem) {
 
     if (notification.target_url && notification.target_url !== '#') {
         targetUrl = new URL(notification.target_url).pathname;
+        $('.ow-message-target-redirect').removeClass('ow-hide');
```

Comment:
Can you please add one new selenium test method only for this specific detail, which is quite important?